.github/ISSUE_TEMPLATE/orientation-epic.md

```diff
@@ -14,30 +14,43 @@ As a new VA.gov team member, I need to complete onboarding activities so I can s
 - [ ] Each team member has completed each item in the orientation checklist below.
 
 ## Onboarding Checklist
+### Share your info with others
+- [ ] Name
+- [ ] Role
+- [ ] Email
+- [ ] Github handle
+- [ ] Start date
+- [ ] VFS Team
 
+### Attend general onboarding
 - [ ] PM: Attend intro meeting with product support PM
```

Comment:
this can be included in the "practice area specific" section(s) below under Product Managers